controllers/controllerTwo.js

```diff
@@ -0,0 +1,7 @@
+const endPointData = require('../endpoints.json');
+
```

Comment:
these files could have more descriptive names eg topicsControllers and apiControllers